tests/upgrade/run.sh

```diff
@@ -401,8 +401,8 @@ deploy_earlier_central() {
     info "Deploying: $EARLIER_TAG..."
 
     mkdir -p "bin/$TEST_HOST_OS"
-    gsutil cp "gs://stackrox-ci/roxctl-$EARLIER_TAG" "bin/$TEST_HOST_OS/roxctl"
-    chmod +x "bin/$TEST_HOST_OS/roxctl"
+    gsutil cp "gs://stackrox-ci/roxctl-$EARLIER_TAG" "bin/${TEST_HOST_OS}_amd64/roxctl"
+    chmod +x "bin/${TEST_HOST_OS}_amd64/roxctl"
     PATH="bin/$TEST_HOST_OS:$PATH" command -v roxctl
```

Comment:
You need to adjust these lines to `${TEST_HOST_OS}_amd64` as well. The problem is that we copy the binary to `linux_amd64`, but we don't add it to the path, so the deploy scripts don't find the roxctl binary and thus assume we need to run docker to invoke roxctl, which doesn't work in CI. This is the cause of the upgrade tests failure.